### ISSUE TITLE
Duplicate configuration steps

### ISSUE THREAD

**Issue by WimVandierendonck** [MicrosoftDocs/OfficeDocs-SharePoint#3604]
Under the heading 'Configure View in File Explorer with Edge' step 2 is a duplicate of step 4. 
Step number 2 can be removed because the current step 3 should be done before the 'ConfigureViewInFileExplorer policy' actions.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 2c3f9bb0-43af-b544-3fd7-71eed410cbec
* Version Independent ID: 250786b4-90f1-0eab-c15d-2f0182d69e5e
* Content: [View SharePoint files with File Explorer in Microsoft Edge - SharePoint in Microsoft 365](https://learn.microsoft.com/en-us/sharepoint/sharepoint-view-in-edge)
* Content Source: [SharePoint/SharePointOnline/sharepoint-view-in-edge.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/public/SharePoint/SharePointOnline/sharepoint-view-in-edge.md)
* Service: **sharepoint-online**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

**Comment by get-itips** [comment]
thanks for your feedback
